docs/source/guide/configuration.rst

```diff
@@ -48,9 +50,83 @@ To set these configuration options, create a ``Config`` object with the options
 
     client = boto3.client('kinesis', config=my_config)
 
+
+Using proxies
+~~~~~~~~~~~~~
+Boto3 supports using proxies as intermediaries between your code and AWS. Proxies may provide any number of functions, from filtering to security and firewalls to privacy assurance.
+
+
+.. _specify_proxies:
+
+Specifying proxy servers
+''''''''''''''''''''''''
+
+You can specify proxy servers to be used when communicating between specific endpoints, or when
+using specific protocols. The ``proxies`` option in the ``Config`` object is a dictionary that maps
+the protocol name or endpoint URL to the address of a proxy server.
+
+.. code-block:: python
+
+    import boto3
+    from botocore.config import Config
+
+    proxy_list = {
+        'http': 'proxy.example.com:6502',
+        'http://mysite.org': 'proxy.example.org:2010'
+    }
+
+    my_config = Config(
+        'region_name': 'us-east-2',
+        'signature_version': 'v4',
+        'proxies': proxy_list
+    }
+
+    client = boto3.client('kinesis', config=my_config)
+
+
+In the above example, a proxy list is set up to use ``proxy.example.com``, port 6502 as the proxy for all HTTP requests by default. For requests to ``http://mysite.org``, port 2010 on ``proxy.example.org`` is used instead.
+
+
+.. _configure_proxies:
+
+Configuring proxies
+'''''''''''''''''''
+You can configure proxy usage with the ``proxies-config`` option, which is a dictionary that specifies the values of several proxy options by name. The valid keys for this dictionary are:
+
+* ``proxy_ca_bundle`` (string) - The pathname of a custom certificate bundle to use when establishing TLS/SSL connections using a proxy.
```

Comment:
I think ideally we keep these in the botocore config page like the s3 config options. Otherwise we have options spread across different locations which may hurt discoverability.

Reply:
My feeling is that you can always make things more discoverable by ensuring that the information is right where the user is looking for it. If they're reading about how to configure proxies, telling them the format of the `proxies_config` class is relevant. Making users flip around through pages to get everything they need isn't very nice. :)

One thing I might be able to do is to use the sphinx/reStructuredText `include` directive to embed the list from the reference documentation...

Reply:
I finally accepted reality and removed the stuff, with a simpler bit of prose replacing it.